pkgs/development/tools/build-managers/buildbot9/plugins.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv,
+  fetchurl,
+  buildPythonApplication,
+  python
+}:
+
+{
+  www = buildPythonApplication rec {
+    name = "buildbot-www-0.9.0rc1";
+    namePrefix = "";
+    format = "wheel";
```

Comment:
what is the reason for building from a wheel? Please build from source when possible.